utils/database.js

```diff
@@ -13,41 +13,40 @@ class Database {
         resolve(true);
         return;
       }
-      db = SQLite.openDatabase({
+      db = SQLite.openDatabase(
+        {
           name: databaseName,
-          createFromLocation: 1
-        }, () => {
-          console.log("Database open Successfully")
+          createFromLocation: 1,
+        },
+        () => {
+          console.log("Database open Successfully");
         },
         (error) => {
-          console.log("Error", error)
-        });
+          console.log("Error", error);
+        }
+      );
       resolve(true);
-    })
+    });
   }
 
   static getBaniList(language) {
-    return new Promise((resolve)=> {
+    return new Promise((resolve) => {
       Database.initDB().then(() => {
-        db.executeSql(
-          "SELECT ID, Gurmukhi, Transliterations FROM Banis",
-          [],
-          (results) => {
-            
-            const totalResults = {};
-            const len = results.rows.length;
+        db.executeSql("SELECT ID, Gurmukhi, Transliterations FROM Banis", [], (results) => {
+          const totalResults = {};
+          const len = results.rows.length;
 
-            // eslint-disable-next-line no-plusplus
-            for (let i = 0; i < len; i++) {
-              const row = results.rows.item(i);
-              totalResults[row.ID] = {
-                gurmukhi: row.Gurmukhi,
-                translit: getTranslitText(row.Transliterations, language),
-              };
-            }
-            resolve(totalResults);
+          // eslint-disable-next-line no-plusplus
```

Comment:
Veerji, Instead of disabling the rule, you can put it like `i += 1` or apply any other good fix. Let me know what's the best approach for you for this.

Ref: https://eslint.org/docs/latest/rules/no-plusplus

Reply:
fixed